src/index.js

```diff
@@ -1,27 +1,46 @@
 import './styles.css';
+import Task from './tasks.js';
+import displayTask from './displayTask.js';
+import { editTask, editText, keyPress } from './editTask.js';
 
-const tasks = [
-  {
-    description: 'Practice leetcode algorithms and data structures',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Afterwards set up for codewars algorithms and data structures',
-    completed: false,
-    index: 2,
-  },
-];
+const newTask = new Task();
+const inputTask = document.querySelector('.input-task');
 
-const todoListContainer = document.querySelector('.todo-list');
+function updateUi(id) {
+  const localData = JSON.parse(localStorage.getItem('tasks'));
+  if (localData !== null) {
+    const deleteIndex = localData.findIndex((item) => item.index === id);
+    localData.splice(deleteIndex, 1);
+  }
+  newTask.tasks = localData;
+  localStorage.setItem('tasks', JSON.stringify(localData));
+}
```

Comment:
- [ ] Kindly in your delete method, when I try to delete the first item, it removes the last item from the local storage, kindly checks to remove the correct index from the local storage.

![image](https://user-images.githubusercontent.com/35535130/160960680-4fe417a3-bd39-4b2e-b4fb-6adf1d4463e6.png)